kiwi/bootloader/config/systemd_boot.py

```diff
@@ -178,7 +178,6 @@ def _run_bootctl(self, root_dir: str) -> None:
                 '--entry-token', 'os-id'
             ]
         )
-        Path.wipe(f'{root_dir}/boot/loader')
```

Comment:
This makes me nervous. Is there a reason we do this in the first place and what makes it okay to drop this?

Reply:
Actually this line should have never existed. When I implemented sdboot support initially I wanted to ignore eventually OS provided entry files. That on its own is not a good move. On further iterations we enhanced the support of sdboot to update existing OS provided entry files when needed and only create our own ones if the distro doesn't provide any (like it's the case on SUSE for example).

We missed to drop the unconditional deletion and I spotted this as a side effect when building read-only images where you simply cannot deleted data that is on the read-only media.

That's why this PR came up. kiwi should never delete data just like that

Reply:
So from my perspective a clear bug that must be fixed. If you want to delete stuff you can put this line to your config.sh and then it becomes a responsibility of the author of the description, which is fine imho

Reply:
Makes sense to me.